frontend/app/views/fragments/form/nameDetailPayPal.scala.html

```diff
@@ -0,0 +1,46 @@
+@(firstName: Option[String], lastName: Option[String])(implicit request: RequestHeader)
+@import utils.TestUsers.testUsers
+
+@if(firstName.exists(testUsers.isValid)) {
+    <!-- VALID TEST USER -->
```

Comment:
No idea, I didn't touch that, it's also empty on the [main page](https://github.com/guardian/membership-frontend/blob/master/frontend/app/views/fragments/form/nameDetail.scala.html).